recipes/planitpoker/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "planitpoker",
+  "name": "PlanITPoker",
+  "version": "1.0.0",
+  "license": "MIT",
+  "config": {
+    "serviceURL": "https://www.planitpoker.com/board/#/room/{teamId}",
```

Comment:
Are all the urls for the teams of that form? If yes, then it would be useful to the user to add
``` "urlInputPrefix": "planitpoker.com/board/#/room/"```
Otherwise, please provide the possibility to set the correct url with an example.

Reply:
Great! I missed this configuration option. Thank you for pointing it out!